db/migrate/20191106121438_unique_index_on_moves.rb

```diff
@@ -0,0 +1,5 @@
+class UniqueIndexOnMoves < ActiveRecord::Migration[5.2]
+  def change
+    add_index :moves, [:from_location_id, :to_location_id, :person_id, :date, :time_due], name: 'index_on_move_uniqueness', unique: true
```

Comment:
Given https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/156#discussion_r343683943, this index should be `:from_location_id, :to_location_id, :person_id, :date`.